containers/troubleshooting.md

```diff
@@ -62,3 +62,7 @@ Possible causes:
 ## <a id="filenotfound"></a>File Not Found Errors
 Possible causes:
   - You are not specifying the full path of the docker config file: please make sure you are using the full path, not the relative path. The file should only include the credentials for the image being certified.
+
+## <a id="callingremoteapierrors"></a>Calling Remote API Errors
+Possible causes:
+  - You are using the old opsid in --certification-project-id, you could find the project id in project link  https://connect.redhat.com/projects/1234567890/overview
```

Comment:
I think the wording here should be consistend with the `--help` output of `preflight` You can see that verbiage [here](https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/main/cmd/check_container.go#L294-L295)